drones/main.py

```diff
@@ -0,0 +1,25 @@
+"""Main project file.
+
+It's where the API entrypoints are accessed and used for the purpose of the project.
+"""
+
+import logging
+from connection import c  # Sample library import

```

Comment:
If you use the suggestions above, here you will be able to import `connector` instance from `prototype.py`.